ustriage/ustriage.py

```diff
@@ -765,12 +774,17 @@ def launch():
     date_range = {'start': args.start_date,
                   'end': args.end_date}
 
+    if args.age is False and (args.show_subscribed or args.show_tagged):
+        args.age = 7
+    if args.age is not False:
```

Comment:
If we drop the `is [not] False` and rely on the truth value of `args.age` this should just work as expected when `args.age` is `0` (as I suggest above), which is falsy.

```
if not args.age and (...):

if args.age:
```

Reply:
As I explained, we need to be able to differentiate "0" from "not set", therefore we can't make the default just 0 and therefore this won't work IMHO.

Reply:
+1